### DISCUSSION TITLE
Popover not translating correctly when using --radix-popover-trigger-width

### DISCUSSION THREAD

**Discussion by itsfaqih** [radix-ui/primitives#2021] [Help] [answered]
I'm not sure if it's from the radix ui or from the styling that caused this. It's all fine if I use fixed width, but not when using the `--radix-popover-trigger-width`

Here's the screenshot of the popover:
![image](https://user-images.githubusercontent.com/48067039/224907976-d9313224-e541-451b-abd2-4b19f19a1d14.png)

You can check out the code here:
https://stackblitz.com/edit/vitejs-vite-iq1caq?file=src/App.tsx

**Comment by benoitgrelard** [comment, marked as answer]
Hey that's been fixed recently. If you install the latest rc (we haven't published a stable yet) it should be working.

**Reply by itsfaqih** [reply]
ahh sorry didn't know that. thanks for telling me

**Reply by benoitgrelard** [reply]
No worries, it isn't advertised anywhere yet so not your fault!
